native/cocos/platform/Image.cpp

```diff
@@ -329,7 +330,10 @@ bool Image::initWithImageData(const unsigned char *data, uint32_t dataLen) {
                 ret = initWithETC2Data(unpackedData, unpackedLen);
```

Comment:
No need to pass `level` parameter to `initWithETCData` and `initWithETC2Data`?

Reply:
Native should support merging of all texture formats，Modify later

Reply:
Only '*.astc', '*.pvr', and '*.pvm' compression textures are included in this revision